qiita_db/metadata_template/test/test_sample_template.py

```diff
@@ -1848,63 +1759,63 @@ def test_extend_new_samples_and_columns(self):
         self.assertItemsEqual(st.categories(), exp_categories)
         exp_dict = {
             "%s.Sample1" % st.id: {
-                'collection_timestamp': datetime(2014, 5, 29, 12, 24, 51),
+                'collection_timestamp': '05/29/14 12:24:15',
```

Comment:
After a second review, I'm wrong but now all tests pass, except the one that I said that requires DB changes.